### DISCUSSION TITLE
"Service is marked for deletion"

### DISCUSSION THREAD

**Discussion by stdedos** [winsiderss/systeminformer#1519] [Questions / Answers] [answered]
I after switching to SystemInformer, I see two things:

* The "error" dialog: "Service is marked for deletion"

But I don't get it - "it looks alright":
```
C:\Windows\system32>sc query KSystemInformer

SERVICE_NAME: KSystemInformer
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 4  RUNNING
                                (STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0
```

I don't like the "Start type: Disable", but that might be natural ... right?
![image](https://user-images.githubusercontent.com/133706/204621829-19c73c4b-37f7-4989-9444-f19c8d02c57a.png)

* On UAC elevation, I am getting "Publisher: Unknown"

"I am accustomed to prompts like this", but, is it going away "nowadays"? 🙏

**Comment by dmex** [comment, marked as answer]
> The "error" dialog: "Service is marked for deletion"

You shouldn't see this unless third party software is messing with the driver.

**Reply by stdedos** [reply]
Same deal I guess with responses I've found around about "somewhat unrelated" topics (or so I thought).

Thank you 🙂
